### ISSUE TITLE
Errbit error on Fedora 23

### ISSUE THREAD

**Issue by RobbiNespu** [Aluxian/Whatsie#209]
Hi there!

* Operating system:
$ cat  /etc/fedora-release
Fedora release 23 (Twenty Three)
$ uname -a
Linux localhost.localdomain 4.4.9-300.fc23.x86_64 #1 SMP Wed May 4 23:56:27 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

* Whatsie version:
> 2.0.14

I getting  this error when launch from terminal 
```
$ whatsie 
[Sun, 29 May 2016 13:38:17 GMT] /scripts/browser/managers/auto-update-manager.js: [Error: Conflict]
reporting to Errbit: { [Error: Conflict]
  url: '/scripts/browser/managers/auto-update-manager.js',
  component: 'whatsie:auto-update-manager',
  session: 
   { distrib: 'linux64:rpm',
     process_type: 'browser',
     electron_version: 'v0.37.6',
     app_version: '2.0.14',
     raw_stack: 
      [ 'Error: Conflict',
        '    at Request._callback (app/scripts/browser/components/auto-updater/impl-linux.js:103:32)',
        '    at Request.self.callback (app/node_modules/request/request.js:200:22)',
        '    at emitTwo (events.js:100:13)',
        '    at Request.emit (events.js:185:7)',
        '    at Request.<anonymous> (app/node_modules/request/request.js:1046:10)',
        '    at emitOne (events.js:95:20)',
        '    at Request.emit (events.js:182:7)',
        '    at IncomingMessage.<anonymous> (app/node_modules/request/request.js:973:12)',
        '    at emitNone (events.js:85:20)',
        '    at IncomingMessage.emit (events.js:179:7)',
        '    at ()',
        '    at app/scripts/browser/utils/logger.js:153:44',
        '    at AutoUpdater.<anonymous> (app/scripts/browser/managers/auto-update-manager.js:135:11)',
        '    at emitOne (events.js:95:20)',
        '    at AutoUpdater.emit (events.js:182:7)',
        '    at Request._callback (app/scripts/browser/components/auto-updater/impl-linux.js:103:18)',
        '    at Request.self.callback (app/node_modules/request/request.js:200:22)',
        '    at emitTwo (events.js:100:13)',
        '    at Request.emit (events.js:185:7)',
        '    at Request.<anonymous> (app/node_modules/request/request.js:1046:10)',
        '    at emitOne (events.js:95:20)' ] } }
reporting to Errbit failed
{ [Error: getaddrinfo ENOTFOUND exceptions.whatsie.chat exceptions.whatsie.chat:443]
  code: 'ENOTFOUND',
  errno: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'exceptions.whatsie.chat',
  host: 'exceptions.whatsie.chat',
  port: 443 
}
```

What this error mean here? Plus, attaching document are not working. The receiver did't receive the document. I check my phone..there is no msg about attached document.

**Comment by flanaras** [comment]
I get the same error when I launch it on openSUSE Leap 42.1, where I also used an rpm package:
`Linux localhost.localdomain 4.1.21-14-default #1 SMP PREEMPT Sun Apr 17 07:27:45 UTC 2016 (fc187c1) x86_64 x86_64 x86_64 GNU/Linux`

If I go "App -> Check for Update" I will invoke the same error again.

**Comment by Aluxian** [comment]
This should be fixed in the latest update (2.0.15).
